esp-wifi/src/wifi/mod.rs

```diff
@@ -3093,6 +3126,98 @@ mod asynch {
                     static WAKER: AtomicWaker = AtomicWaker::new();
                     &WAKER
                 }
+                WifiEvent::ConnectionlessModuleWakeIntervalStart => {
+                    static WAKER: AtomicWaker = AtomicWaker::new();
```

Comment:
We're starting to waste a bit more memory here than I'm comfortable with. Should we have a single waker and accept that waiting for an event will potentially be polled a bit more than optimal?

Reply:
I guess that would be fine ... events are not emitted in very high frequency 👍